core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.ts

```diff
@@ -26,13 +26,21 @@ import { PaginatedResultEvent, ResultExportResponse } from "../../types/workflow
 import { NotificationService } from "../../../common/service/notification/notification.service";
 import { ExecuteWorkflowService } from "../execute-workflow/execute-workflow.service";
 import { ExecutionState, isNotInExecution } from "../../types/execute-workflow.interface";
-import { filter } from "rxjs/operators";
+import { catchError, filter, map, take, tap } from "rxjs/operators";
 import { OperatorResultService, WorkflowResultService } from "../workflow-result/workflow-result.service";
 import { DownloadService } from "../../../dashboard/service/user/download/download.service";
 import { HttpResponse } from "@angular/common/http";
 import { ExportWorkflowJsonResponse } from "../../../dashboard/service/user/download/download.service";
 import { DashboardWorkflowComputingUnit } from "../../types/workflow-computing-unit";
 import { GuiConfigService } from "../../../common/service/gui-config.service";
+import { DatasetService } from "../../../dashboard/service/user/dataset/dataset.service";
+import { parseFilePathToDatasetFile } from "../../../common/type/dataset-file";
+
+export interface RestrictionAnalysisResult {
+  restrictedOperatorMap: Map<string, Set<string>>;
```

Comment:
Rename to `WorkflowResultDownloadability`. Add comment on each field to list the specification